Source/Mosa.Kernel.BareMetal/Debug.cs

```diff
@@ -244,3 +253,21 @@ private static void WriteValueDigit(ulong val, int position, byte @base)
 
 	#endregion Private API
 }
+
+public struct Hex
+{
+	public ulong Value;
+
+	public Hex(ulong value) => Value = value;
+
+	public Hex(Pointer value) => Value = value.ToUInt64();
+}
+
+public struct Hex8
+{
+	public ulong Value;
+
+	public Hex8(ulong value) => Value = value;
+
+	public Hex8(Pointer value) => Value = value.ToUInt64();
+}
```

Comment:
Those 2 structs seem identical?

Reply:
Almost, it's a different type. The type determines that output formatting.